src/api/app/models/package.rb

```diff
@@ -1386,25 +1386,28 @@ def api_obj
     self
   end
 
+  #### WARNING: these operations run in build object, not this package object
   def rebuild(params)
-    backend_build_command(:rebuild, params.slice(:package, :arch, :repository))
+    backend_build_command(:rebuild, params[:project], params.slice(:package, :arch, :repository))
```

Comment:
Oh and why don't we slice :project too and just use it in `backend_build_command`?